setup.cfg

```diff
@@ -36,9 +36,11 @@ where = src
 test =
     pytest
     pytest-asyncio
+    pytest-mock
     responses
     aioresponses
     tornado
+    katsdptelstate[aio]
```

Comment:
Should we not have katsdptelstate as an extras_require in setup.py ? I don't see how a non-test install would actually install katsdptelstate given the current state of setup and requirements.

Reply:
I considered it, but my feeling is that it doesn't any value because the only way to use the functionality is to already have a telstate object (or something that quacks like one), so there is no way for there to be a missing requirement.

Reply:
Or to phrase it another way: katsdpmodels *always* supports TelescopeStateFetcher, rather than it being an optional extra.